src/diffusers/models/attention_processor.py

```diff
@@ -284,8 +284,9 @@ def __init__(
                 self.norm_added_q = RMSNorm(dim_head, eps=eps)
                 self.norm_added_k = RMSNorm(dim_head, eps=eps)
             elif qk_norm == "rms_norm_across_heads":
-                # Wanx applies qk norm across all heads
-                self.norm_added_q = RMSNorm(dim_head * heads, eps=eps)
+                # Wan applies qk norm across all heads
+                # Wan also doesn't apply a q norm
+                self.norm_added_q = None
```

Comment:
@yiyixuxu Discovered an issue when running the slow test for the transformer. The diffusers implementation has this extra `norm_added_q` key which the original does not. When converting from original checkpoint there is not weight to assign to this norm, so it remain a meta tensor, so we run into an error when setting the model to a device. 

Removing this, and then adding `norm_added_q` to the `_keys_to_ignore_on_load_unexpected` in the transformer so that the warning about extra keys in the Diffusers version is suppressed. 

Ideal solution is to update the weights in the model repo, but that could take time. This is a fix for the meantime

Reply:
sounds good!